packages/twenty-front/src/modules/ui/navigation/menu-item/internals/components/StyledMenuItemBase.tsx

```diff
@@ -25,7 +25,7 @@ export const StyledMenuItemBase = styled.div<MenuItemBaseProps>`
 
   font-size: ${({ theme }) => theme.font.size.sm};
 
-  gap: ${({ theme }) => theme.spacing(2)};
+  gap: ${({ theme }) => theme.spacing(6.4)};
```

Comment:
this does not feel right :) What's the root cause here?

Reply:
The root cause is actually wrapping in UndecoratedLink, adding a prop to require fullWidth